Cargo.toml

```diff
@@ -1,10 +1,12 @@
+cargo-features = ["resolver"]
+
 [workspace]
 resolver = "2"
 
 members = [
 
-	"bin/zero-dev/node",
-	"bin/zero-dev/runtime",
+	"bin/zero/runtime",
```

Comment:
you can keep both, dev and full node, maybe for the next iteration we will move selection of which to build to parameters

Reply:
I'd suggest to use both once we are able to specify needed runtime as parameter and skip it until then. Otherwise build will include both versions and will require more time to finish. It already takes decent amount of time for build to finish so no reason to make it run longer :laughing: